test/manual/webpack.config.js

```diff
@@ -17,9 +19,12 @@ module.exports = {
   },
   plugins: [
     new Self({
-      filename: '[name].css',
-      chunkFilename: '[contenthash].css',
+      filename: '[name].client.css',
+      chunkFilename: '[name].chunk.client.css',
+      globalRTLFlag: 'rtlLanguageEnabled',
+      // outputPublicPath: '/bundles/css/',
```

Comment:
```suggestion
```

Reply:
This only pertains to the testing environment. Decided to keep this comment for testing later if things need to be adjusted or if the fork needs to pull in changes from the master origin.